src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/eventDecoder.hpp

```diff
@@ -71,6 +73,10 @@ class EventDecoder final : public AbstractHandler<std::shared_ptr<EventContext>>
             {
                 data->resourceType = ResourceType::CNA_MAPPING;
             }
+            else if (Utils::startsWith(data->resource.at("resource"), "ADP-DESCRIPTION-GLOBAL"))
```

Comment:
Question: shouldn't the name `ADP-DESCRIPTION-GLOBAL` match `base-rules/adp-global.json`  file name?